### PR TITLE
Merge upstream ansel1/merry

### PR DESCRIPTION
**Pull request by gir**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
 
 go:
-  - 1.5.4
-  - 1.6.2
-  - 1.7.4
+  - 1.5
+  - 1.6
+  - 1.7
   - 1.8
+  - 1.9
```
Comment (johnpfeifferhelix) [inline .travis.yml:8]: nit: there's go 1.10  =)

**Review by brian-at-helix — APPROVED** [review]

**Comment by johnpfeifferhelix** [comment]
After this merge was completed and downstream consumers (i.e. everybody) uses it could we just start leveraging/import merry directly (in glide.yaml/lock?) instead of an identical helix/hss version of merry?

**Review by johnpfeifferhelix — APPROVED** [review]

**Review by wangdrew — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (gir) [reply .travis.yml:8]: I'm just merging from the upstream =)

**Comment by gir** [comment]
@johnpfeifferhelix I'm not sure the context around the 2 additional merges in this fork and if PRs to the original merry should be made. I'm just updating this fork.

**Pull request merged**